src/main/scala/ru/org/codingteam/horta/protocol/jabber/MucMessageHandler.scala

```diff
@@ -102,9 +102,9 @@ class MucMessageHandler(locale: LocaleDefinition,
       case None =>
         log.warning(s"Cannot find participant $jid in the participant list")
         CommonAccess
-      case Some(participant) => participant.affiliation match {
+      case Some(p) => p.affiliation match {
```

Comment:
My dear friend, shortening variables helps to get rid of compiler / IDE warnings (this longer name was shadowing class member in this case). I think these thingies are worth editing when you spot one.